### PR TITLE
Solved Form Validation Issue #29

### PR DESCRIPTION
**Pull request by Spurthi7768**
Address #29 
For address line 1 I have removed supernumeric method and for city I have created new method called lettersandspace. Please go through the code and let me know if there are any changes.

### CODE REVIEW

File: js/editPairForm.js [hunk 1]
```diff
@@ -385,14 +389,15 @@ $(document).ready(function () {
...
-          supernumeric: true
+          //supernumeric: true
         },
         d_addr2: {
           supernumeric: true
         },
         d_city: {
-          nowhitespace: true,
-          lettersonly: true,
+          lettersandspace:true
```
Comment (siv2r) [inline js/editPairForm.js:398]: Instead of creating a new function(i.e, `lettersandspace`). Can you reuse the `supernumeric` method here?

**Comment by Spurthi7768** [comment]
I have tried to reuse the supernumeric method. Can you check the code once again?

File: js/editPairForm.js [hunk 2]
```diff
@@ -242,7 +242,7 @@ $(document).ready(function () {
 
     // Custom validation methods
     $.validator.addMethod("supernumeric", function (value, element) {
-      return this.optional(element) || /^[A-Za-z0-9_,-.' ]+$/.test(value);
+      return this.optional(element) || /^[A-Za-z0-9_,-.' /s ]+$/.test(value);
```
Comment (siv2r) [inline js/editPairForm.js:245]: no need to add a `/s` here since ` ` (space is included after `'`) is present in the regex.

Comment (siv2r) [reply js/editPairForm.js:245]: Remain all looks fine!! Just make this change and I will merge it :)

**Review by siv2r — CHANGES_REQUESTED** [review]

**Comment by Spurthi7768** [comment]
@siv2r I have removed the space character. Sorry for the trouble caused!

**Comment by siv2r** [comment]
@Spurthi7768 There was no trouble caused!! This is the process :)

**Pull request merged**